lib/Runtime/Base/JnDirectFields.h

```diff
@@ -685,7 +685,7 @@ ENTRY(builtInJavascriptObjectEntryIsExtensible)
 ENTRY(builtInJavascriptObjectEntryKeys)
 ENTRY(builtInJavascriptObjectGetOwnPropertyDescriptor)
 ENTRY(builtInJavascriptObjectPreventExtensions)
-ENTRY(builtInJavascriptRegExpEntryTest)
+ENTRY(builtInJavascriptRegExpEntryTest) // TODO(jahorto): is this needed?
```

Comment:
>// TODO(jahorto): is this needed? [](start = 40, length = 33)

is it?

Reply:
I don't think its used right now, but I want to keep it because _.test might be faster than _.match sometimes, which isn't something we should forget about.